### ISSUE TITLE
From normalize: Line-height should be 1.5 to comply the WCAG guideline

### ISSUE THREAD

**Issue by thomashockaday** [ikrishg/reseter.css#147]
Original issue in Normalize: https://github.com/necolas/normalize.css/issues/864

Particularly the bit about Turkish characters is interesting.

The guideline in question: https://www.w3.org/WAI/WCAG21/Understanding/text-spacing.html

Currently using 1.15 in this repo (https://github.com/resetercss/reseter.css/blob/main/css/reseter.css#L10)

**Comment by GaryHilares** [comment]
Just as an update, it seems like the right link where the `line-height` property is being used now is [this one](https://github.com/krshkun/gardevoir/blob/main/src/components/text.scss).

**Comment by gptkrsh** [comment]
That line-height not for the normal text, it is for the `sub` and `super` script:

https://github.com/krshoss/gardevoir/blob/main/src/components/text.scss#L14-L19

But this is there in: https://github.com/krshoss/gardevoir/blob/main/src/components/global.scss#LL10

If anyone is interested for contributing please change the 
```css
line-height: 1.15
```

with

```css
line-height: 1.5
```

Marking this as a gfi
